doc/releases.md

```diff
@@ -147,8 +147,7 @@ Commits may need to be reverted or a major version bump may need to happen.
 
 #### Step 1: Collecting the formatted list of changes:
 
-Collect a formatted list of commits since the last release. Use
-[`changelog-maker`](https://github.com/rvagg/changelog-maker) to do this.
+Collect a formatted list of commits since the last release. Use [`changelog-maker`](https://github.com/rvagg/changelog-maker) to do this:
```

Comment:
I'm pretty sure this will fail linting because lines need to be wrapped at 80 characters.